jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/modal/ModalLiveDemoTests.java

```diff
@@ -0,0 +1,94 @@
+package io.github.epam.bootstrap.tests.composite.section.modal;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.modalLiveDemo;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.testng.Assert.assertEquals;
+
+
+public class ModalLiveDemoTests extends TestsInit {
+
+    private String launchButtonText = "Launch demo modal";
+    private String titleText = "MODAL TITLE";
+    private String bodyText = "Woohoo, you're reading this text in a modal!";
+    private String saveButtonText = "Save changes";
+    private String closeButtonText = "Close";
+
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @Test
+    public void modalContentGetTextTest() {
+        assertEquals(modalLiveDemo.launchModal.getText(), launchButtonText);
+        modalLiveDemo.launchModal.click();
+        assertEquals(modalLiveDemo.modalContent.title.getText(), titleText);
+        assertEquals(modalLiveDemo.body.getText(), bodyText);
+        assertEquals(modalLiveDemo.saveButton.getText(), saveButtonText);
+        assertEquals(modalLiveDemo.closeButton.getText(), closeButtonText);
```

Comment:
You should use JDI asserts instead od TestNG asserts.

Reply:
As far as i know, sometimes it's ok to use both testNG and JDI asserts. For example, we can see it in Roman's tests which are standart if there are any doubts about our own tests:
https://github.com/jdi-testing/jdi-light/blob/bootstrap/jdi-light-html-tests/src/test/java/io/github/epam/html/tests/elements/common/TextTests.java

Reply:
It is another type of tests, he use them when test elements themselves. Something like unit tests.
For examlpe, there is one of tests by your link:
```
@Test
    public void getTextTest() {
        assertEquals(jdiText.getText(), text);
    }
```
It doesn't test text in element, it test method getText().

It is not your case, because you shouldn't test method getText() of Button there, in  ModalLiveDemoTests. You should show, how to use JDI to test text in Button in such type of modal.